src/allmydata/test/test_abbreviate.py

```diff
@@ -0,0 +1,147 @@
+"""
+Tests for allmydata.util.abbreviate.
+"""
```

Comment:
This docstring should declare the module has been ported to Python 3 right?

Reply:
Yes, fixed.